cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -6,9 +6,13 @@
 import subprocess
 import webbrowser
 import sys
+import requests
+
+from gidgethub.sansio import create_headers
```

Comment:
Not important, but you shifted from importing modules to importing objects with this import line.